src/OpenCL.cpp

```diff
@@ -187,8 +187,8 @@ void __in_transform_eq(float x[4][4], __global float * restrict V, int offset, i
 __kernel void in_transform(__global net_t * restrict in, __global float * restrict V,
                            const int C, const int Cpad,
                            const int Ppad) {
-    const int W = 19;
-    const int H = 19;
+    const int W = BOARD_SIZE;
```

Comment:
consider adding BOARD_SQUARES = BOARD_SIZE*BOARD_SIZE;
which can be substituted in dozens of places here.